specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -240,29 +255,14 @@
           },
           {
             "$ref": "#/parameters/SourceControlIdParameter"
-          },
-          {
-            "$ref": "#/parameters/SourceControl"
           }
         ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/SourceControl"
-            }
-          },
-          "201": {
-            "description": "Created",
-            "schema": {
-              "$ref": "#/definitions/SourceControl"
-            }
+            "description": "OK"
```

Comment:
200 Response is never without response schema.

Reply:
Fixed.